### PR TITLE
Home improvements 

### PR DESCRIPTION
**Pull request by dhakelila**
-Ranking table linked to API.
-The second slide added to the slider. 
-Text added to locale file. 
-Hide "latest activity" table until we'll have data. (It is commented on the code, but please, keep it because we will use it.)

Note: There is a react warning that we need to fix, It was being there for all the time and it doesn't seem to avoid the app from working but it needs to be fixed anyway.

### CODE REVIEW

**Comment by simaob** [comment]
Let's round the number under the "Join the Community" # of Roads.

File: config/locales/en.yml [hunk 1]
```diff
@@ -68,3 +68,9 @@ en:
         step_two_legend: Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam. dolor sit amet, consectetur adipisicing elit.
         step_three_title: MAP THE ROADS
         step_three_legend: Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam. dolor sit amet, consectetur adipisicing elit.
+        step_four: GO TO A TASK
```
Comment (simaob) [inline config/locales/en.yml]: nest under `to_fix` and start from one (they are independent steps).

File: app/assets/javascripts/main.js [hunk 2]
```diff
@@ -64,21 +64,21 @@ $('#communityData').ready( function() {
...
-      ReactDOM.render(
-        <Provider store={store}>
-          <UsersActivityDataContainer data={element} />
-        </Provider>,
-        document.getElementById(element)
-      );
-    });
-  }
-});
+// $('#latestActivity').ready( function() {
```
Comment (simaob) [inline app/assets/javascripts/main.js]: I saw the comment in the PR about this bit of code being commented. But I think it's useful if you leave a comment here explaining why it's not removed.

**Comment by dhakelila** [comment]
Fixed.

**Pull request merged**